src/main/kotlin/Calculator.kt

```diff
@@ -0,0 +1,58 @@
+import java.util.LinkedList
+import java.util.Queue
+
+object Calculator {
+
+    fun calEquation(equation: String?): Double {
+        if (equation == null || equation.isEmpty()) {
```

Comment:
`String?`에 대해 호출할 수 있는 `isNullOrEmpty()` 또는 `isNullOrBlank()`가 있습니다.

```suggestion
        if (equation.isNullOrBlank()) {
```

Reply:
적용하였습니다.
[4fef0dc02cf80daa59d0e7026d0c8e2c7b034a38]